src/index.html

```diff
@@ -11,7 +11,171 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-box">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <lable>
+            Surname:
+            <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            >
+          </lable>
+        </div>
+
+        <div class="form-field">
+          <lable>
+            Name:
+            <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            >
+          </lable>
+        </div>
```

Comment:
```suggestion
        <div class="form-field">
          <lable>
            Surname:
            <input
              type="text"
              name="surname"
              autocomplete="off"
            >
          </lable>
        </div>

        <div class="form-field">
          <lable>
            Name:
            <input
              type="text"
              name="name"
              autocomplete="off"
            >
          </lable>
        </div>
```
check code style, fix all such cases

Reply:
checklist 
<img width="911" alt="image" src="https://user-images.githubusercontent.com/102411130/218665409-8cd3dfe6-5943-4794-a75e-46a8eba1f544.png">